### ISSUE TITLE
CELL Lighting changes not propagated

### ISSUE THREAD

**Issue by cmeeren** [matortheeternal/smash#211]
As can be seen in the screenshot below, cell lighting (`XCLL`, but also Image Space `XCIM` and Lighting Tempate `LTMP` not visible on the screenshot) is not forwarded. For this patch all plugins except `MLU.esp` was set to `Smash.All` whereas `MLU.esp` was set to `Smash.ForceAll`.

![image](https://user-images.githubusercontent.com/7766733/42733968-cd4400f4-883b-11e8-9869-877261f4a039.png)

**Comment by matortheeternal** [comment]
> except `MLU.esp` was set to Smash.All whereas `MLU.esp` was set to Smash.ForceAll.

Typo?  Which was it set to?

**Comment by cmeeren** [comment]
No typo, read again.

**Comment by matortheeternal** [comment]
OK, that sentence was throwing me for a loop.

So there's no problem here.  Smash is doing exactly what you told it to do.  ForceAll means that the record will be forced to the version in MLU.esp, and Smash will ignore conflicts in plugins loaded after it unless the plugin requires MLU.esp as a master.

**Comment by cmeeren** [comment]
Thanks. Sorry for the knotty phrasing.

Apart from modifying `ELE_SSE.esp` to include `MLU.esp` as a master (I have no idea what else this will do), is there any way I can include the lighting change from ELE? What happens if I set `ELE_SSE.esp` to `Smash.ForceAll` – will the `FULL` change from `MLU.esp` be overwritten with the one from `ELE_SSE.esp` (which is identical to master)?

Currently I have used Wrye Bash for merging the leveled lists, and have set `MLU.esp` to `Smash.All`. Not sure of the full consequences of this.
